### PR TITLE
PS-6115 [Doc] Add step to Encryption Threads (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
added note to upgrading guide with link to encryption threads doc
added section to encryption threads doc

### CODE REVIEW

File: doc/source/security/encrypting-threads.rst
Comment (robgolebiowski) [inline doc/source/security/encrypting-threads.rst]: s/turn on encryption/turn on encryption threads/

File: doc/source/security/encrypting-threads.rst
Comment (robgolebiowski) [inline doc/source/security/encrypting-threads.rst]: You have to explicitly set ENCRYPTION='N' for tablespace to be excluded from encryption by encryption threads.

File: doc/source/security/encrypting-threads.rst
Comment (robgolebiowski) [inline doc/source/security/encrypting-threads.rst]: This is called implicit ENCRYPTION='N' and does not cause tablespace to be excluded from encryption threads. Only explicit ENCRYPTION='N' causes tablespace to be excluded from encryption threads. This might be bit confusing - so the best way to check which tablespaces are excluded from encryption threads is to check INFORMATION_SCHEMA.INNODB_TABLESPACES_ENCRYPTION. If the table/tablespace you want to excluded is not in this table (and have field EXCLUDED set to 'Y'), issue ALTER TABLE/TABLESPACE ENCRYPTION='N'.

File: doc/source/upgrading_guide.rst
Comment (robgolebiowski) [inline doc/source/upgrading_guide.rst]: s/INFORMATION_SCHEMA_ENCRYPTION/INFORMATION_SCHEMA.INNODB_TABLESPACES_ENCRYPTION/

Comment (robgolebiowski) [reply doc/source/upgrading_guide.rst]: The tablespaces that are excluded from encryption threads have 'Y' in EXCLUDED column.

File: doc/source/upgrading_guide.rst
Comment (robgolebiowski) [inline doc/source/upgrading_guide.rst]: Here we also need to distinguish between explicit ENCRYPTION='N' and implicit ENCRYPTION='N'

**Review by robgolebiowski — CHANGES_REQUESTED** [review]

File: doc/source/security/encrypting-threads.rst
Comment (patrickbirch) [reply doc/source/security/encrypting-threads.rst]: added

File: doc/source/security/encrypting-threads.rst
Comment (patrickbirch) [reply doc/source/security/encrypting-threads.rst]: Updated.

File: doc/source/security/encrypting-threads.rst
Comment (patrickbirch) [reply doc/source/security/encrypting-threads.rst]: Updated.

File: doc/source/upgrading_guide.rst
Comment (patrickbirch) [reply doc/source/upgrading_guide.rst]: Rewritten.

File: doc/source/upgrading_guide.rst
Comment (patrickbirch) [reply doc/source/upgrading_guide.rst]: Rewritten.

**Comment by patrickbirch** [comment]
Waiting for either approval or comments.

File: doc/source/security/encrypting-threads.rst
Comment (robgolebiowski) [reply doc/source/security/encrypting-threads.rst]: The upgrade scenario between 5.7 and 8.0 should be removed for encryption threads. This is because we have decided to remove encryption threads from 5.7.

**Review by robgolebiowski — CHANGES_REQUESTED** [review]

File: doc/source/security/encrypting-threads.rst
Comment (patrickbirch) [reply doc/source/security/encrypting-threads.rst]: Removed.

**Comment by patrickbirch** [comment]
Updated.

**Review by robgolebiowski — APPROVED** [review]

**Pull request closed without merging**
